pkgs/applications/editors/emacs/macport_noescape_noop.h

```diff
@@ -0,0 +1,31 @@
+#ifndef NOESCAPE_NOOP_H_
```

Comment:
Can this file be upstreamed?

Reply:
They use Apple Clang I believe which presumably behaves differently.